internal/kafka/internal/routes/route_loader.go

```diff
@@ -86,7 +87,11 @@ func (s *options) buildApiBaseRouter(mainRouter *mux.Router, basePath string, op
 
 	authorizeMiddleware := s.AccessControlListMiddleware.Authorize
 	requireOrgID := auth.NewRequireOrgIDMiddleware().RequireOrgID(errors.ErrorUnauthenticated)
-	requireIssuer := auth.NewRequireIssuerMiddleware().RequireIssuer([]string{s.ServerConfig.TokenIssuerURL}, errors.ErrorUnauthenticated)
+	issuerList := []string{s.ServerConfig.TokenIssuerURL}
+	if s.Keycloak.GetConfig().SelectSSOProvider == keycloak.REDHAT_SSO {
+		issuerList = append(issuerList, s.Keycloak.GetRealmConfig().TokenEndpointURI)
```

Comment:
```suggestion
		issuerList = append(issuerList, s.Keycloak.GetRealmConfig().ValidIssuerURI)
```

`ValidIssuerURI` or `TokenEndpointURI`?